website/static/_redirects

```diff
@@ -14,6 +14,70 @@
 /contributing/how-to-contribute         /contributing/overview
 /releases/release-candidate-minor         /releases/release-branch-cut-and-rc0
 
+# Move New Architecture docs of all versions
+/docs/the-new-architecture/why                             /docs/next/the-new-architecture/landing-page
+/docs/the-new-architecture/use-app-template                https://github.com/reactwg/react-native-new-architecture#guides
+/docs/the-new-architecture/pillars                         https://github.com/reactwg/react-native-new-architecture#guides
+/docs/the-new-architecture/pillars-turbomodules            https://github.com/reactwg/react-native-new-architecture#guides
+/docs/the-new-architecture/pillars-fabric-components       https://github.com/reactwg/react-native-new-architecture#guides
+/docs/the-new-architecture/pillars-codegen                 https://github.com/reactwg/react-native-new-architecture#guides
+/docs/the-new-architecture/cxx-cxxturbomodules             https://github.com/reactwg/react-native-new-architecture#guides
+/docs/the-new-architecture/the-new-architecture/cxx-custom-types https://github.com/reactwg/react-native-new-architecture#guides
+/docs/the-new-architecture/backward-compatibility          https://github.com/reactwg/react-native-new-architecture#guides
+/docs/the-new-architecture/backward-compatibility-turbomodules https://github.com/reactwg/react-native-new-architecture#guides
+/docs/the-new-architecture/backward-compatibility-fabric-components https://github.com/reactwg/react-native-new-architecture#guides
+/docs/new-architecture-intro                               https://github.com/reactwg/react-native-new-architecture#guides
+/docs/new-architecture-library-intro                       https://github.com/reactwg/react-native-new-architecture#guides
+/docs/new-architecture-library-android                     https://github.com/reactwg/react-native-new-architecture#guides
+/docs/new-architecture-library-ios                         https://github.com/reactwg/react-native-new-architecture#guides
+/docs/new-architecture-app-intro                           https://github.com/reactwg/react-native-new-architecture#guides
+/docs/react-18-and-react-native                            https://github.com/reactwg/react-native-new-architecture#guides
+/docs/new-architecture-troubleshooting                     https://github.com/reactwg/react-native-new-architecture#guides
+/docs/new-architecture-appendix                            https://github.com/reactwg/react-native-new-architecture#guides
+# We do explicit versions because Netlify placeholders dont match periods
+# https://answers.netlify.com/t/placeholder-redirects-dont-support-periods/113543
+/docs/0.72/the-new-architecture/landing-page                    /docs/next/the-new-architecture/landing-page
```

Comment:
What's really odd is that these don't work, even when when I explicitly write them out 🤔
See related PR: https://github.com/facebook/react-native-website/pull/4029

Reply:
Hmm, It's only a guess, but maybe you need also to remove the pages from versioned sidebars and versioned docs, so the Docusuarus do not try to overwrite or modify the redirects in any way? 

* https://github.com/facebook/react-native-website/tree/main/website/versioned_sidebars
* https://github.com/facebook/react-native-website/tree/main/website/versioned_docs

Looks like this PR only updates the "next" sidebar, and do not touch pages. Spotted also that we lack redirect for 0.70 docs (not all avaible version are deployed in previews, to optimize the build time.

Reply:
Hmm let me try that.

Ah good catch. I'll update the other non-next sidebars as well and I guess the relevant intro page

Reply:
I don't think it's the matter of deleting the docs ... also I think in the end I don't want to delete right now because we do link to them in some blog posts. Can probably do a clean-up job one day.

I also thought it might be this build step around updating _redirects https://github.com/facebook/react-native-website/pull/3680 but inspecting the _redirects file generated from `yarn build`, I don't see how that affects the version. 

I also tried deleting the `$LATEST_VERSION$` redirect. Seriously confused 😵

I think for now, I'll go ahead and land. 0.72 docs will somehow always point to the old New Architecture docs 🤷‍♀️